core/utils/util.c

```diff
@@ -69,7 +69,7 @@ print_message_function_t* print_message_function = NULL;
 /** The level of messages to redirect to print_message_function. */
 int print_message_level = -1;
 
-int lf_fed_id() { return _lf_my_fed_id; }
+int32_t lf_fed_id() { return _lf_my_fed_id; }
```

Comment:
```suggestion
uint16_t lf_fed_id() { return _lf_my_fed_id; }
```

Reply:
`lf_fed_id()` function in `util.c` and `util.h` is actually not used anywhere. `_lf_my_fed_id` is directly use everywhere, so this function can be deleted, however I just left it just in case.

Reply:
I think `lf_fed_id()` is meant to be user facing, part of the API a reactor writer can use. I think it is better to have a function here than unrestricted access to a global variable.